src/view/pipeline/app/common/ModalStructure.tsx

```diff
@@ -0,0 +1,62 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
```

Comment:
It seems that copied file, you need to add comment from which repository you copied all this files.